styleguide/source/_patterns/05-pages/press-release.md

```diff
@@ -2,3 +2,11 @@
 Press page with two columns.
 
 [EWL-4479 SG2 | Create Press release page ](https://issues.ama-assn.org/browse/EWL-4479)
+
+### Pattern Contains
+
+* Tools List - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-tools-block
+* Membership with CTA - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-membership-with-cta
+* Paragraphs - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=atoms-paragraph
+* Tags - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-tags
+* Four up Teaser - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-four-up-teaser
```

Comment:
This is probably actually redundant since PatternLab automatically includes this information in the bottom of the Pattern info. We probably just need a description of the page here. In the Zeplin Toph has the following for that:
Use Case
For all Press Release and Official statements of the AMA

User Case
Quickly learn the actions and statements of the AMA in order to write or publish content

@chelsiejohnston I'd be interested in your thoughts on this as well.